compiler-rt/lib/tsan/rtl/tsan_rtl_access.cpp

```diff
@@ -578,8 +578,11 @@ static void MemoryRangeSet(uptr addr, uptr size, RawShadow val) {
       Die();
   }
   // Set the ending.
-  if (mid2 < end)
+  if (mid2 < end) {
     ShadowSet(mid2, end, val);
+  } else {
```

Comment:
Nit: https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements